src/timecorrelations.jl

```diff
@@ -62,6 +62,39 @@ function correlation(rho0::Operator, H::AbstractOperator, J,
 end
 
 
+"""
+    timecorrelations.correlation_dynamic(tspan, rho0, f, op1, op2; <keyword arguments>)
```

Comment:
```suggestion
    timecorrelations.correlation_dynamic(tspan, rho0, f, A, B; <keyword arguments>)
```

The docstring below mentions ``⟨A(t)B(0)⟩``, while the signature shows `op1, op2`. Please make the two consistent.

Reply:
maybe the same should be applied to `timecorrelations.correlation` as well to match the docstring